src/renderers/dom/test/__tests__/ReactTestUtils-test.js

```diff
@@ -844,4 +844,20 @@ describe('ReactTestUtils', () => {
       );
     });
   });
+
+  it('should call setState with the correct parameters', () => {
+    let mockArg = 'the wrong value';
+    let mockCallback = arg => (mockArg = arg);
```

Comment:
Maybe let's assert `arguments.length` is `0` instead of checking first argument?